### ISSUE TITLE
FEAT: Manual commits and rollbacks

### ISSUE THREAD

**Issue by Tarrowren** [beekeeper-studio/beekeeper-studio#1036]
It makes me nervous every time I execute sql statements.
I wish there was a manual commit and rollback feature, like PLSQL Developer and DBeaver.
![image](https://user-images.githubusercontent.com/35626141/153984597-cddfbb3e-7005-402f-86bb-dcaa3ea5589a.png)

**Comment by rathboma** [comment]
Yeah I hear what you are saying. I think there should be a way to do it, but I'm concerned about making the UI really hard to use.

This isn't a priority right now, but I'll accept it.

**Comment by rathboma** [comment]
Note to future self: Think about a way to do this that makes sense. Maybe add a notification bar at the top which makes it clear that all operations are within a transaction.

**Comment by antoinetielbeke** [comment]
+1 for this feature. I love the manual commit from Dbbeaver and it makes me feel more 'safe' running complex queries.

**Comment by paulipotter** [comment]
+1 here too. The company I work for got licenses for BKS but I can't use it to its fullest because the internal team I work with requires manual commits

**Comment by ScripterSugar** [comment]
This is so important feature for most db jobs. +1 for here too, I'm about to refund and migrate to another alternative just because the lack of this feature... I really love beekeeper's aesthetics but lacking such crucial protection cannot be tolerated.

**Comment by honestlysamuk** [comment]
+1 from me too. Similar to https://github.com/beekeeper-studio/beekeeper-studio/issues/3565
